Workbooks/WorkbooksMetadata.json

```diff
@@ -420,10 +420,10 @@
     "logoFileName": "trendmicro_logo.svg",
     "description": "Gain insights into your Trend Micro Deep Security security event data by visualizing your Deep Security Anti-Malware, Firewall, Integrity Monitoring, Intrusion Prevention, Log Inspection, and Web Reputation event data.",
     "dataTypesDependencies": [ "TrendMicroDeepSecurity" ],
-    "dataConnectorsDependencies": [ "TrendMicro" ],
+    "dataConnectorsDependencies": [ "TrendMicroDeepSecurity" ],
```

Comment:
This change is not needed since the data connector ID won't change (refer to the comment on the if for data connector).